src/gstrtpreceiver.cpp

```diff
@@ -78,13 +82,18 @@ GstRtpReceiver::GstRtpReceiver(int udp_port, const VideoCodec& codec)
     m_port=udp_port;
     m_video_codec=codec;
     initGstreamerOrThrow();
-
+    splash_enabled = config["splash"]["enabled"].as<bool>(false);
+    splash_file    = config["splash"]["file"].as<std::string>("");
+    splash_timeout = config["splash"]["timeout"].as<int>(3);
```

Comment:
might be better to pass splash config as arguments rather than extracting them from `yaml`. This is to not "contaminate" the whole codebase with configuration file details. IMO it's better to extract all the data from YAML config in the `main.cpp:main` function - this way we can detect config errors at early stage rather than crash somewhere deep